lms/envs/common.py

```diff
@@ -5211,6 +5211,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 WIKI_HELP_URL = "https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_assets/course_wiki.html"
 CUSTOM_PAGES_HELP_URL = "https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_assets/pages.html#adding-custom-pages"
 COURSE_BULK_EMAIL_HELP_URL = "https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/manage_live_course/bulk_email.html"
+OPEN_RESPONSES_HELP_URL = "https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_assets/pages.html#enabling-the-flexible-peer-grade-for-ora"
```

Comment:
Does this need to be set twice, here in cms/envs/common.py and in lms/envs/common.py? 

In any case, this setting needs annotation as required by OEP-17: https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0017-bp-feature-toggles.html

Reply:
The environment need to exist for both `lms` and `cms`. `CUSTOM_PAGES_HELP_URL` also using this twice as well. It would be nice to have something like `common/envs/common.py` to share the environment variable but that doesn't exist at the moment.